frontstage/views/sign_in/sign_in.py

```diff
@@ -47,12 +47,13 @@ def login():
             oauth2_token = oauth_controller.sign_in(username, password)
         except OAuth2Error as exc:
             error_message = exc.oauth2_error
-            if BAD_AUTH_ERROR in error_message:
-                return render_template('sign-in/sign-in.html', form=form, data={"error": {"type": "failed"}})
-            elif USER_ACCOUNT_LOCKED in error_message:
+            if USER_ACCOUNT_LOCKED in error_message:
                 logger.info('User account is locked on the OAuth2 server')
                 if party_json['status'] == 'ACTIVE' or party_json['status'] == 'CREATED':
                     change_respondent_status(respondent_id=party_id, status='SUSPENDED')
+                notify_account_locked(username=username)
```

Comment:
Do you want to send an email every time a user tries to log in with a locked account? Probably should have this within the `if party_json['status'] == 'ACTIVE'` block so it only done when the account is set to suspended.

Possibly even have it as logic in party (when setting status to suspended send out email to user) so you don't have to call party twice and have it as part of a single transaction.